fontbe/src/features.rs

```diff
@@ -2,7 +2,7 @@
 
 use std::path::{Path, PathBuf};
 
-use fea_rs::{Diagnostic, GlyphMap, GlyphName};
+use fea_rs::{Diagnostic, GlyphMap, GlyphName as FeaRsGlyphName};
```

Comment:
I assume we'll want to pull `fea-rs` into this repo eventually and unify these types somehow.

Reply:
IMHO we should have a common GlyphName both consume from somewhere else. Tracking in https://github.com/googlefonts/fontations/issues/201.